qiskit/aqua/algorithms/minimum_eigen_solvers/qpe.py

```diff
@@ -86,7 +86,7 @@ def __init__(self,
         validate_min('num_ancillae', num_ancillae, 1)
         validate_in_set('expansion_mode', expansion_mode, {'trotter', 'suzuki'})
         validate_min('expansion_order', expansion_order, 1)
-        super().__init__(quantum_instance)
+        super().__init__(quantum_instance)  # type: ignore
```

Comment:
I see that in some files you don't have the addition of a `warn_class` call. Is that because it is handled by `super()`?

Reply:
QPE derives from QuantumAlgorithm and MinimumEigensolver. It didn't complain before because MinimumEigensolver had no init declared but since I added one there with the deprecated msg, mypy complains now, that is why I disabled it: qiskit/aqua/algorithms/minimum_eigen_solvers/qpe.py:89: error: Too many arguments for "__init__" of "MinimumEigensolver". It is a wrong msg because the order of init calls follows the order of the inheritance at the top: first QuantumAlgorithm init with its argument and then MinimumEigensolver automatically without arguments.